### ISSUE TITLE
"for example" Vs. "e.g." in 3.3.7 and 3.3.8 Benefits

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2375]
I know we had a small discussion in the past about the use of "for example" Vs. "e.g." and have chosen to use one of them consistently, not sure which one was chosen, but 3.3.7 and 3.3.8 both have a different approach...

3.3.7
https://w3c.github.io/wcag/understanding/accessible-authentication.html

> Benefits
> People with cognitive issues relating to memory, reading (**for example**, dyslexia), numbers (**for example**, dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.

3.3.8
https://w3c.github.io/wcag/understanding/accessible-authentication-no-exception.html

> Benefits
> The benefits of this success criterion are similar to Accessible Authentication.
> 
> People with cognitive issues relating to memory, reading (**e.g.** dyslexia), numbers (**e.g.** dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.

**Comment by awkawk** [comment]
Also note that when we use e.g. there is a comma following it (e.g., https://www.w3.org/TR/WCAG21/#cc2, https://www.w3.org/TR/WCAG21/#dfn-assistive-technologies).

That said, we are missing the comma in:
* note in 1.4.10
* 2nd bullet in 2.1.4
* the 3rd note in 5.2.2
* 4 times in "style property" definition

We use it correctly 18 times in the normative document and incorrectly seven times.

**Comment by bruce-usab** [comment]
From the discussions I recall the consensus was to use <q>for example</q> going forward as it is plain language.  I do not recall if we memorialized that.

**Comment by awkawk** [comment]
Is "e.g." not plain language? I get that it is a latin abbreviation, but I would think that it is common enough to be widely understood.

**Comment by patrickhlauke** [comment]
+1 for expanded "for example", "for instance", and so on. "e.g." may not be fully understood by international audiences (anecdotally anyway)

**Comment by alastc** [comment]
The "e.g."s in the [understanding doc](https://w3c.github.io/wcag/understanding/accessible-authentication-no-exception.html) have been updated. I don't think AWK was suggesting we change all the normative examples, so I'll close this issue.
